### ISSUE TITLE
White March Part 1

### ISSUE THREAD

**Issue by Xaratas** [phob/pillarsofeternity-german-patch#171]
Was werden wir vorfinden? Backer-Content-Qualität oder Praktikantenarbeit?

**Comment by Xaratas** [comment]
Die Übersetzung der Extension hat einen eigenen Ordner erhalten. Ein eigenes repository oder beide mischen?

**Comment by AurelioSilver** [comment]
Da bin ich überfragt, Eigender Branch? Aber ka, wie das dann funktioniert.

**Comment by Xaratas** [comment]
Hab mit meinem Kollegen darüber gesprochen, das einfachste ein neues Git, das beide Ordner enthalten kann.

**Comment by phob** [comment]
Werden die Änderungen jetzt noch hier rein gespielt oder habt ihr die White March Updates nun nur im neuen Repository?

**Comment by AurelioSilver** [comment]
Vermutlich nur im neuen. Ist einfacher. Wobei ich die Änderungen, die sich auf Version ohne Expansion beziehen, weitergepflegt habe. Nicht jeder hat die Erweiterung. Ich z.B. :)

Wobei vermutlich man sich auch die Expansiontexte runterladen könnte, ohne dass es das Spiel ohne Erweiterung irgendwie beinflußt.

**Comment by Xaratas** [comment]
Ich hab die 2 Gits ineinander verschachtelt vorliegen und kann jederzeit in dieses hier pushen. Denke aber, dass nur das neue zu verwenden auf Dauer einfacher ist.

**Comment by phob** [comment]
Die git-Splittung kann ich nicht wirklich nachvollziehen weil das hätte man auch alles hier machen können, da es ja nur ein neuer Ordner ist der dazu gekommen ist und es hier ein simples Script zum erzeugen der Nexus-Mod gibt, aber gut.

**Comment by Xaratas** [comment]
Ja, aber der neue Ordner ist 2 Ebenen weiter oben angesiedelt, sicher hätte man das mit Hardlinks lösen können (Softlinks werden von Git als Dateien betrachtet), hätte aber die Git Mitarbeit von anderen erschwert.
Oder man hätte große verschiebe commits ausgeführt, dann hätten aber auch alle die das git schon haben es neu auschecken müssen. Ein neues Git war da nach meiner Meinung die eleganteste Lösung.
